Configurations/unix-Makefile.tmpl

```diff
@@ -1113,6 +1113,40 @@ generate_doc_buildinfo:
                 mv $(SRCDIR)/doc/build.info.new $(SRCDIR)/doc/build.info; \
           fi )
 
+generate_fips_sources: $(SRCDIR)/providers/fips.module.sources
+$(SRCDIR)/providers/fips.module.sources: \
+                $(SRCDIR)/Configure \
+                {- join(" \\\n" . ' ' x 16,
+                        fill_lines(" ", $COLUMNS - 16,
+                                   @{$config{build_file_templates}},
+                                   @{$config{build_infos}},
+                                   @{$config{conf_files}})) -}
+	rm -rf sources-tmp
+	mkdir sources-tmp
+	( \
+	  srcdir=`cd $(SRCDIR); pwd`; \
+	  cd sources-tmp \
+	  && $$srcdir/Configure \
+	  && ./configdata.pm --query 'get_sources("providers/fips")' > sources1 \
+	  && $$srcdir/Configure no-asm \
```

Comment:
This will be broken I suppose if we make fips disabled by default. Please add explicit enable-fips.

Reply:
Good point.  Fixed

Reply:
This still requires adding `enable-fips` as a Configure argument otherwise when fips is no longer enabled by default sources for the fips provider will not be generated, or am I wrong?

Reply:
Correct.  I forgot we discussed making that non-default...

Reply:
Fixed that too now